settings/common.py

```diff
@@ -198,3 +198,10 @@
 SITE_PROTOCOL = 'http'
 
 MARKDOWN_EXTENSIONS = ['linkify']
+
+
+# twitter settings
+CONSUMER_KEY = os.environ.get('CONSUMER_KEY', '')
```

Comment:
@ChillarAnand can we have it prefixed with `TWITTER` so it's not confusing later.